contracts/RelayHub.sol

```diff
@@ -211,7 +211,7 @@ contract RelayHub is IRelayHub {
             return uint256(PreconditionCheck.WrongSignature);
         }
 
-        // Verify the transaction is not being repalyed
+        // Verify the transaction is not being replayed
```

Comment:
If you're at it, there is one in line 387: responsability -> responsibility :+1: